app/src/main/java/com/kickstarter/ui/fragments/projectpage/ProjectCampaignFragment.kt

```diff
@@ -66,7 +79,28 @@ class ProjectCampaignFragment :
             .compose(bindToLifecycle())
             .compose(Transformers.observeForUI())
             .subscribe {
-                binding?.projectCampaignViewListItems?.smoothScrollToPosition(it)
+                binding?.projectCampaignViewListItems?.smoothScrollToPosition(it + 1)
+            }
+
+        this.viewModel.outputs.onOpenVideoInFullScreen()
+            .subscribeOn(Schedulers.io())
+            .distinctUntilChanged()
+            .compose(bindToLifecycle())
+            .compose(Transformers.observeForUI())
+            .subscribe {
+                val intent = Intent(context, VideoActivity::class.java)
```

Comment:
Can you move the code to open the video activity as an extension to `ActivityExt` ?